pyriemann/utils/covariance.py

```diff
@@ -30,6 +30,46 @@ def _mcd(X):
     return C
 
 
+def _sch(X):
+    """Schaefer-Strimmer covariance estimator
+
+    Shrinkage estimator using method from [1]:
+    .. math::
+            \hat{\Sigma} = (1 - \gamma)\Sigma_{scm} + \gamma T
+
+    where :math:`T` is the diagonal target matrix:
+    .. math::
+            T_{i,j} = \{ \Sigma_{scm}^{ii} \text{if} i = j, 0 \text{otherwise} \}
+    Note that the optimal :math:`\gamma` is estimate by the authors' method.
+
+    :param X: Signal matrix, (n_channels, n_trials)
```

Comment:
The shape seems to be (n_trials, n_channels, n_times)...

Reply:
I think there is no problem. This  function needs (n_channels, n_times) input. The (n_trials, n_channels, n_times) input is for `covariances(X, estimator='cov')`, see [here](https://github.com/pyRiemann/pyRiemann/blob/master/pyriemann/utils/covariance.py#L97).

Reply:
Yes, you are right!